core/primitives/address/impl/address_builder_impl.cpp

```diff
@@ -6,28 +6,31 @@
 #include "primitives/address/impl/address_builder_impl.hpp"
 
 #include "crypto/blake2/blake2b160.hpp"
-#include "primitives/address_codec.hpp"
+#include "primitives/address/address_codec.hpp"
 
 namespace fc::primitives::address {
 
-  using primitives::Address;
+  using primitives::address::Address;
   using Sec256k1PublicKey = libp2p::crypto::secp256k1::PublicKey;
   using BlsPublicKey = crypto::bls::PublicKey;
   using crypto::blake2b::blake2b_160;
+  using primitives::address::decode;
+  using primitives::address::encode;
+  using primitives::address::Protocol;
 
   outcome::result<Address> AddressBuilderImpl::makeFromSecp256k1PublicKey(
       Network network, const Sec256k1PublicKey &public_key) noexcept {
     OUTCOME_TRY(hash, blake2b_160(public_key));
-    std::vector<uint8_t> sec256k1_bytes{network, primitives::SECP256K1};
+    std::vector<uint8_t> sec256k1_bytes{network, Protocol::SECP256K1};
```

Comment:
not an issue, just to know. why Secp256k1 is all capitalized?

Reply:
BLS is ok since it is an acronym

Reply:
it is also an acronym) anyway, if memory serves me right, we decided to use upper-case in constants. sometimes it applies to enums members, but it is not fully legal.